src/firstboot/firstboot.c

```diff
@@ -592,7 +607,7 @@ static int prompt_timezone(int rfd) {
         print_welcome(rfd);
 
         r = prompt_loop("Please enter timezone name or number",
-                        zones, 30, timezone_is_valid_log_debug, &arg_timezone);
+                        zones, "UTC", 30, timezone_is_valid_log_debug, &arg_timezone);
```

Comment:
I am not entirely happy with this. Would it make sense to instead of hardcoding the compile time values to query the current value and set based on that? so here `get_timezone` and what would the one for locale be?

This should then in theory default to the compile time values unless they have already been manually changed, in which case it can somewhat safely be assumed that that value is more likely to be wanted.